packages/expo-dev-menu/ios/EXDevMenuBuildInfo.m

```diff
@@ -3,6 +3,25 @@
 
 @implementation EXDevMenuBuildInfo
 
+// TODO -- EXManifest - use actual interface
+//- (nullable NSString *)sdkVersion;
+//- (NSString *)bundleUrl;
+//- (nullable NSString *)revisionId;
+//- (nullable NSString *)slug;
+//- (nullable NSString *)appKey;
+//- (nullable NSString *)name;
+//- (nullable NSString *)version;
+//- (nullable NSDictionary *)notificationPreferences;
+//- (nullable NSDictionary *)updatesInfo;
+//- (nullable NSDictionary *)iosConfig;
+//- (nullable NSString *)hostUri;
+//- (nullable NSString *)orientation;
+//- (nullable NSDictionary *)experiments;
+//- (nullable NSDictionary *)developer;
+//- (nullable NSString *)facebookAppId;
+//- (nullable NSString *)facebookApplicationName;
+//- (BOOL)facebookAutoInitEnabled;
+
 +(NSDictionary *)getBuildInfoForBridge:(RCTBridge *)bridge andManifest:(NSDictionary *)manifest
```

Comment:
it looks like we are a bit inconsistent with the terms "BuildInfo" and "AppInfo" -- i wonder if it would be helpful to use the two to differentiate between the info that comes from the manifest (which i would consider to be the "app" info) and the info that is baked into the build (which i would consider to be the "build" info).

Reply:
those names definitely make sense, but I'm a bit hesitant to make any distinctions like this - I would prefer to call everything `appInfo` and not worry at all about where it comes from, I think its making things overly complex at this point

Reply:
in that case, do you think it would make sense to standardize all the variable/method names to one or the other?

Reply:
yup - I'll update the variable/method names to `AppInfo`